requirements.txt

```diff
@@ -1,11 +1,11 @@
 Flask>=1.0,<1.1
-stpmex>=1.0.0
+stpmex>=2.0.4
```

Comment:
make sure it's less than `3.0.0`. Otherwise, a breaking change to the library would also break `speid`